src/applications/vaos/tests/new-appointment/components/TypeOfVisitPage.unit.spec.jsx

```diff
@@ -34,59 +34,62 @@ describe('VAOS <TypeOfVisitPage> ', () => {
 
     expect(screen.getAllByRole('radio').length).to.equal(3);
   });
-});
-
-it('should not submit empty form', async () => {
-  const store = createTestStore(initialState);
-  const screen = renderWithStoreAndRouter(<TypeOfVisitPage />, {
-    store,
-  });
-  expect(await screen.findByText(/Continue/i)).to.exist;
 
-  fireEvent.click(screen.getByText(/Continue/));
+  it('should not submit empty form', async () => {
```

Comment:
I copied and pasted the screener page tests from this file, and realized that the describe block was wrapping all the tests. So I fixed that. These changes are just indentation adjustments.